File: 미누_문민우/week6/js/Main.js
class User {
    constructor(name, email, phone) {
        this.name = name;
        this.email = email;
        this.phone = phone;
    }
    printInfo(){
        let str;
        str = "이름: " + this.name + '\n'
            + "이메일: " + this.email + '\n'
            + "휴대폰번호: " + this.phone;

        console.log(str);
    }
}

const strArray = ['Hello', 'Hi', 'Bye'];

for(let i = 0; i < strArray.length; i++) {
    console.log(toUpperCase(strArray[i]));
}
function toUpperCase(str) {
    str = str.toUpperCase();
    return str;
}

console.log("=====================================")

const user = new User("미누", "test1234@gmail.com", "010-1234-5678");
user.printInfo();

console.log("=====================================")

const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on("line", (line) => {
    try {
        const num = 0;
        if(num === 0) {
            throw new Error("Cannot divide by zero");
        } else {
            console.log(line / num);
        }

    } catch (e) {
        console.log(e.message);
    }

    rl.close();
});

rl.on('close', () => {
    process.exit();
})

File: 미누_문민우/week6/java/Main.java
package week6.java;

import java.util.Scanner;

class Student {

    private String name;
    private String studentID;
    private String major;

    public Student(String name, String studentID, String major) {
        this.name = name;
        this.studentID = studentID;
        this.major = major;
    }

    public void printInfo() {

        String studentInfo;
        studentInfo = "이름: " + this.name + '\n'
                + "학번: " + this.studentID + '\n'
                + "전공: " + this.major;

        System.out.println(studentInfo);
    }
}

public class Main {
    public static void main(String[] args) {

        Student student = new Student("미누", "12345678", "컴퓨터공학과");
        student.printInfo();

        System.out.println("===================================");

        System.out.print("0 ~ 10 even number is ");
        for (int i = 1; i <= 10; i++) {
            if (isEven(i)) {
                System.out.print(i + " ");
            }
        }

        System.out.println();
        System.out.println("===================================");

        System.out.print("Enter the divide number: ");
        Scanner scanner = new Scanner(System.in);
        int divide = scanner.nextInt();

        try {
            System.out.println(divide / 10);
            System.out.println(divide / 0);

        } catch (ArithmeticException e) {
            System.out.println("Cannot divide by zero");
        }
    }

    public static boolean isEven(int num) {
        return num % 2 == 0;
    }
}
